weave

```diff
@@ -1105,6 +1104,59 @@ DOCKER_VERSION_PATCH=$(echo "$DOCKER_VERSION" | cut -d. -f 3)
 COMMAND=$1
 shift 1
 
+deprecation_warning() {
```

Comment:
They don't take arguments, but do output a deprecation warning, so we should use this for consistency.